tests/TypeTest.php

```diff
@@ -33,7 +33,7 @@ public function test_success($type, $value)
      * @test
      * @dataProvider failProvider
      */
-    public function test_fail($type, $value)
+    public function fail($type, $value)
```

Comment:
Ha, this fails because of [`PHPUnit\Framework\Assert::fail()`](https://github.com/sebastianbergmann/phpunit/blob/7.1.5/src/Framework/Assert.php#L2616-L2628).

Would you prefer to 

a) rename the conflicting test
b) consistently use the `test_` prefix instead of the `@test` annotation

?

Reply:
Oh let's just rename the test :) We should always use `@test` and not the `test_` prefix anymore!